### DISCUSSION TITLE
does INDICATE work at BlueZ

### DISCUSSION THREAD

**Discussion by Guesel** [hbldh/bleak#529] [Q&A]
* bleak version: 0.11.0
* Python version: 3.8.7
* Operating System: Raspbian GNU/Linux 10 (buster)
* BlueZ version (`bluetoothctl -v`) in case of Linux: 5.54

### Description

For a Blood Pressure Device i try to read the measurement value. The characteristic "2A35" depends on the INDICATION.
I wrote an simple routine  with client.start_notify(char_uuid, callback). But no event is detected.
For an Heartratemonitor with NOTIFY the routine works well.
It looks like INDICATION does not work for BlueZ. Is it true? At bleak/backends/bluezdbus/client.py i does'nt find any hint to INDICATION implementation.
Have I something overlooked?

**Comment by dlech** [comment]
As mentioned in https://github.com/hbldh/bleak/issues/526#issuecomment-826107556, if a single characteristic has property flags set for both notify and indicate, BlueZ will prefer notify. If the characteristic only has the property flag for indications, it should work.
